aries-site/src/pages/components/dateinput.mdx

```diff
@@ -0,0 +1,70 @@
+import { Example } from '../../layouts';
+import {
+  DateInputRangeExample,
+  DateInputExample,
+  DateInputSimpleExample,
+  DateInputValidationExample,
+} from '../../examples/components/dateinput';
+
+
+<Example
+  docs="https://v2.grommet.io/dateinput?theme=hpe"
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/dateinput/DateInputSimpleExample.js"
+  figma="https://www.figma.com/file/Mf9if6zyGETTWW5b9PsYNQ/HPE-Date-Input-Component?node-id=1%3A567"
+>
+  <DateInputSimpleExample />
+</Example>
+
+## Guidance
+
+- DateInput can be used to display past, present, or future dates and times.
+- DateInput is an intuitive way for users to input a date or time by providing them with a familiar calendar interface.
```

Comment:
I think we can also say that DateInput provides a more guided experience for collecting dates from the user rather than a plain TextInput. The `format` provides MaskedInput constraints on what can be typed and auto-fills fixed characters. The Calendar Drop provides a familiar visual grid interface.